rpc/client/mock/client.go

```diff
@@ -29,15 +28,7 @@ import (
 
 // Client wraps arbitrary implementations of the various interfaces.
```

Comment:
nit: change the godoc?

Reply:
I'd be happy to, but I think what it says here is still as correct now as it was before. Do you have a suggestion for what might be better here?